qiskit/test/mock/backends/fake_100q/defs_100q.json

```diff
@@ -0,0 +1,592 @@
+{
+    "buffer": 10,
+    "cmd_def": [
```

Comment:
We require a full basis gate set for all qubits and CX gates to connect them. These could be copied from a preexisting backend.